src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -754,7 +754,7 @@ function MoneyRequestView({
                                             upgradePath: CONST.UPGRADE_PATHS.CATEGORIES,
                                         }),
                                     );
-                                } else if (shouldSelectPolicy) {
+                                } else if (!isPolicyExpenseChat && shouldSelectPolicy) {
```

Comment:
```suggestion
                                } else if (!policy && shouldSelectPolicy) {
```

Reply:
Makes more sense, thank you for catching that!